logstash-core/lib/logstash/modules/importer.rb

```diff
@@ -0,0 +1,39 @@
+# encoding: utf-8
+require "logstash/namespace"
+require "logstash/logging"
+
+module LogStash module Modules class Importer
+  include LogStash::Util::Loggable
+
+  attr_reader :client
```

Comment:
Nice catch!
Removable I think. Used during coding. We should not leak a reference to this, allowing it use outside of the Importer API